test/Readme.md

```diff
@@ -0,0 +1,56 @@
+# Jupiter Broadcasting Site End to End Tests
+
+## Testing Methodology
+These tests are setup using the [Playwright](https://playwright.dev/python/docs/writing-tests) framework. They are intended to run on each commit to verify the site is working in expected order.s
+
+These tests exist to make sure code changes to this repository do not break how the site currently function. 
+
+On every commit, these tests will be run against the commited code. The tests will verify that the site is in proper working order with respect to the changes the commiter has changed.
+
+The reason for this is so that one can feel more confient that their changes will not cause issues on site when it is deployed in production. The goal is to catch these issues well before they make it to production thus not causing headaches for anyone maintaining the site.
```

Comment:
```suggestion
The reason for this is so that one can feel more confident that their changes will not cause issues to the site when it is deployed in production. The goal is to catch these issues well before they make it to production thus not causing headaches for anyone maintaining the site.
```